cheatsheets/Cross_Site_Scripting_Prevention_Cheat_Sheet.md

```diff
@@ -1,400 +1,207 @@
-# Cross Site Scripting Prevention Cheat Sheet
+# Cross Site Scripting (XSS) Prevention Cheat Sheet
 
 ## Introduction
 
-This article provides a simple positive model for preventing [XSS](https://owasp.org/www-community/attacks/xss/) using output encoding properly. While there are a huge number of XSS attack vectors, following a few simple rules can completely defend against this serious attack.
+This cheat sheet provides guidance to prevent XSS vulnerabilities. 
 
-This article does not explore the technical or business impact of XSS. Suffice it to say that it can lead to an attacker gaining the ability to do anything a victim can do through their browser.
+Cross-Site Scripting is a misnomer. The name originated from early versions of the attack where stealing data cross-site was the primary focus. Since then, it has extended to include injection of basically any content, but we still refer to this as XSS. 
 
-Both [reflected and stored XSS](https://owasp.org/www-community/attacks/xss/#stored-and-reflected-xss-attacks) can be addressed by performing the appropriate validation and encoding on the server-side. [DOM Based XSS](https://owasp.org/www-community/attacks/DOM_Based_XSS) can be addressed with a special subset of rules described in the [DOM based XSS Prevention Cheat Sheet](DOM_based_XSS_Prevention_Cheat_Sheet.md).
+XSS is serious and can lead to account impersonation, taking actions as a user, observing user behaviour, loading external content, and stealing sensitive data. This cheatsheet is a list of techniques to prevent or limit the impact of XSS. No single technique will solve XSS. Using the right combination of Framework Security Measures, Output Encoding, and HTML Sanitization will prevent most XSS attacks.
```

Comment:
```suggestion
XSS is serious and can lead to account impersonation, taking actions as a user, observing user behaviour, loading external content, and stealing sensitive data. This cheatsheet is a list of techniques to prevent or limit the impact of XSS. No single technique will solve XSS. Using the right combination of Framework Security Measures, Output Encoding, and HTML Sanitization techniques will prevent most XSS attacks.
```